.goreleaser.yml

```diff
@@ -11,6 +11,13 @@ builds:
   - CGO_ENABLED=0
   ldflags:
   - -s -w -X github.com/gitpod-io/leeway/pkg/leeway.Version={{.Version}}-{{.ShortCommit}}
+  goos:
+    - linux
+    - darwin
+  goarch:
+    - "386"
```

Comment:
I reckon we can drop this one (who's still running a 32bit CPU?)

I'm not 💯 sure leeway even functions properly on darwin - maybe we should remove that entirely.